src/validation.h

```diff
@@ -654,6 +647,8 @@ class CChainState
     void PruneAndFlush();
 
     /**
+     * Find the best known block, and make it the tip of the block chain
+     *
```

Comment:
a375714f24b85b6f9ce7e1765a1137e3b85af009  Should this comment be preserved?
```
* May not be called with cs_main held. May not be called in a		
* validationinterface callback.
 ```

Reply:
Unsure, perhaps @jamesob can help expound on this point a little?

Reply:
It is preserved, at the bottom of this comment.